docs/pages/installation.mdx

```diff
@@ -137,7 +137,7 @@ chart.
 
 The Teleport package in Homebrew is not maintained by Teleport and we can't
 guarantee its reliability or security. We recommend the use of our [official
-Teleport packages](https://goteleport.com/teleport/download?os=mac).
+Teleport packages](https://goteleport.com/download).
```

Comment:
```suggestion
Teleport packages](https://goteleport.com/download/?os=mac#install-links).
```

This link works for me.

Reply:
The link works just fine, but it doesn't actually select the OS; it still seems to pick the selected OS based on my browsers user agent string.

Reply:
Yeah, I see that it's always taken from the user agent and there's no other way to influence that.

https://github.com/gravitational/next/blob/2a335f3fa3a9c84833f13e01b6218247ab92a230/apps/next/pages/download/index.page.tsx#L75

Reply:
🤷🏽‍♂️ 

I'm not sure how critical deep-linking OS releases is across OSes; I feel like for the most important applications where we really care, API clients are discovering releases and downloading or linking directly to build artifacts.